tests/api/Jalasoft.TeamUp.Projects.API.Tests/GetProjectsTest.cs

```diff
@@ -0,0 +1,104 @@
+﻿namespace Jalasoft.TeamUp.Projects.API.Tests
+{
+    using System;
+    using System.Collections.Generic;
+    using Jalasoft.TeamUp.Projects.API.Controllers;
+    using Jalasoft.TeamUp.Projects.Core.Interfaces;
+    using Jalasoft.TeamUp.Projects.Models;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using Moq;
+    using Xunit;
+
+    public class GetProjectsTest
+    {
+        private readonly Mock<IProjectsService> mockService;
+        private readonly Mock<HttpRequest> mockRequest;
+        private readonly GetProjects getResumes;
+
+        public GetProjectsTest()
+        {
+            this.mockService = new Mock<IProjectsService>();
+            this.mockRequest = new Mock<HttpRequest>();
+            this.getResumes = new GetProjects(this.mockService.Object);
+        }
+
+        [Fact]
+        public void GetProjects_Returns_OkObjectResult()
+        {
+            Assert.IsType<OkObjectResult>(this.getResumes.Run(this.mockRequest.Object));
+        }
+
+        [Fact]
```

Comment:
we do not need all of this unit tests, please reach out to @rodrigo-baldiviezo-jalafund to check on this he has an example on testing Get All resumes API

Reply:
@Raul-Gamarra-jalafund  The names were changed in the plural and the test was modified according to what was agreed